### PR TITLE
Feature/coefficients

### PR DESCRIPTION
**Pull request by JakubWijata**
adding a method for the calculation of coefficients

### CODE REVIEW

File: statistics/coefficients.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+    delta_currency = calculate_deltas(data_for_currency_1['rates'])
+    coefficients_result = CoefficientsResult()
+    coefficients_result.median=stats.median(delta_currency)
+    coefficients_result.mode=stats.mode(delta_currency)
+    coefficients_result.stDeviation=stats.pstdev(delta_currency)
+    coefficients_result.coefficientVariation= coefficients_result.stDeviation/stats.mean(delta_currency)*100
+
+    return coefficients_result
+
```
Comment (WiktoriaRozanska) [inline statistics/coefficients.py:19]: In this place, we should have two line spaces before function :D

File: statistics/coefficients.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    delta_currency = []
+    for index, rate in enumerate(rates):
+        if index == 0:
+            continue
+
+        delta = float(rate['mid']) - float(rates[index - 1]["mid"])
+        delta_currency.append(delta)
+
+    return delta_currency
+
```
Comment (WiktoriaRozanska) [inline statistics/coefficients.py:30]: Same

**Review by WiktoriaRozanska — APPROVED** [review]
Overall, good job

**Review by ProjectAntZ — APPROVED** [review]
Gj :)

**Pull request merged**